src/plugins/wizard/public/application/components/data_tab/config_panel.scss

```diff
@@ -0,0 +1,9 @@
+.wizConfigPanel {
+  background: #f0f1f3;
```

Comment:
Should this come from the theme?

Reply:
We currently dont have this value (or similar) in EUI. The UI is still being updated. Will address all hardcoded colors in the fit and finish issue when we go over the styles for the whole app. We will either have to add these values in EUI or update our styles to follow EUI.